integration-cli/docker_cli_config_test.go

```diff
@@ -46,46 +47,34 @@ func (s *DockerSuite) TestConfigHttpHeader(c *check.C) {
 	}`
 
 	err = ioutil.WriteFile(tmpCfg, []byte(data), 0600)
-	if err != nil {
-		c.Fatalf("Err creating file(%s): %v", tmpCfg, err)
-	}
+	c.Assert(err, checker.IsNil)
 
 	cmd := exec.Command(dockerBinary, "-H="+server.URL[7:], "ps")
 	out, _, _ := runCommandWithOutput(cmd)
```

Comment:
Not sure because we are calling the *fake* docker host that is setup in the test (using `server`). So I'm pretty sure the command itself fails but we don't really care about it, we just want to see if it has set up the headers — at least that's my understanding of the test :stuck_out_tongue: .